app/views/components/gds/exportsInputTextWithPrefix.scala.html

```diff
@@ -44,6 +44,20 @@
 }
 
 
+@hint(hintText: Option[String], prefixText: Option[String]) = {
```

Comment:
prefixText was the hidden, accessible message - e.g. "Enter the amount in GBP"

This is now used as an optional "extra" bit of information to the visual hint text.